tests/date/test_getters.py

```diff
@@ -61,6 +61,9 @@ def test_week_of_month():
     assert pendulum.date(2020, 1, 1).week_of_month == 1
     assert pendulum.date(2020, 1, 7).week_of_month == 2
     assert pendulum.date(2020, 1, 14).week_of_month == 3
+    assert pendulum.date(2023, 1, 1).week_of_month == 1
```

Comment:
This is not correct, January 1st 2023 belongs to the last week of December of 2022. Since this is the edge case, it should return 0.